mobile/lib/widgets/backup/album_info_list_tile.dart

```diff
@@ -27,6 +35,84 @@ class AlbumInfoListTile extends HookConsumerWidget {
         .watch(appSettingsServiceProvider)
         .getSetting(AppSettingsEnum.syncAlbums);
 
+    final isDarkTheme = context.isDarkTheme;
+    
+    final previewAsset = useState<Asset?>(null);
+    getPreviewAsset() async {
```

Comment:
Can we extract this logic using a helper method to avoid duplicated code?

Reply:
Agree, but wasn't sure where best to extract to. Where do you recommend for these bits?

Reply:
Nm, consolidated into a single file (there was enough duplication of business logic as well as display logic to justify this imo).